src/main/java/Duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package Duke;
+
+import Duke.Commands.*;
+
+/**
+ * Represent a parser class that handles user commands.
+ */
+public class Parser {
+    /**
+     * An enum to store the specific commands which the user can use.
+     */
+    enum CommandWords {
+        TODO,
+        DEADLINE,
+        EVENT,
+        BYE,
+        DELETE,
+        LIST,
+        MARK,
+        UNMARK,
+        FIND
+    }
+
+    /**
+     * Read user input to return a command to be executed.
+     *
+     * @param input Input by the user.
+     * @param ui Ui object to interact with user.
+     * @param taskList TaskList object that records the task.
+     * @param storage Storage object that saves and loads information.
+     * @return Command A specific command to execute a certain action.
+     */
+    public static Command parse(String input, Ui ui, TaskList taskList, Storage storage) throws DukeException {
+        try {
+            String[] inputs = input.split(" ");
+            CommandWords commandWord = CommandWords.valueOf(inputs[0].toUpperCase());
```

Comment:
I like that the variable here is correctly named in singular form to indicate that it is a word, in contrast with the class name which is plural.